### PR TITLE
Added option to specify both secret key and certs in the same file

### PR DESCRIPTION
**Pull request by MaxF12**
This brings it in line with ssh.load_cert_chain().

### CODE REVIEW

File: src/aioquic/quic/configuration.py [hunk 1]
```diff
@@ -103,7 +105,12 @@ def load_cert_chain(
         Load a private key and the corresponding certificate.
         """
         with open(certfile, "rb") as fp:
-            certificates = load_pem_x509_certificates(fp.read())
+            boundary = b"\n-----BEGIN PRIVATE KEY-----\n"
+            chunks = split(boundary, fp.read())
+            certificates = load_pem_x509_certificates(chunks[0])
+            if len(chunks) == 2:
+                private_key = b"-----BEGIN PRIVATE KEY-----\n" + chunks[1]
```
Comment (jlaine) [inline src/aioquic/quic/configuration.py:112]: Why not re-use `boundary` here?

File: src/aioquic/quic/configuration.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 from dataclasses import dataclass, field
 from os import PathLike
+from re import split
 from typing import Any, List, Optional, TextIO, Union
 
+
```
Comment (jlaine) [inline src/aioquic/quic/configuration.py:6]: Remove this extra blank line

**Comment by jlaine** [comment]
Don't worry about the "package-wheel" CI failures, I'm working on fixing these.

**Comment by MaxF12** [comment]
Should be done, not sure why codecov is missing a line in client.py now.

File: tests/utils.py [hunk 3]
```diff
@@ -53,6 +53,7 @@ def run(coro):
     os.path.dirname(__file__), "ssl_cert_with_chain.pem"
 )
 SERVER_KEYFILE = os.path.join(os.path.dirname(__file__), "ssl_key.pem")
+SERVER_COMBINED = os.path.join(os.path.dirname(__file__), "cert_key.pem")
```
Comment (jlaine) [inline tests/utils.py:56]: Minor nit: `SERVER_COMBINEDFILE` for the variable and `ssl_combined.pem` for the file

**Comment by jlaine** [comment]
Hopefully in a couple of minutes we'll have CI back up, I've had to switch off binary wheels for pypy on OSX. I'll get back to you when you can rebase your PR.

**Comment by jlaine** [comment]
@MaxF12 if you could please fix that final nitpick and rebase on top of `main` we should be good to go!

**Comment by MaxF12** [comment]
Done, Thanks!

**Pull request merged**